distributed/tests/test_client.py

```diff
@@ -6293,27 +6294,18 @@ async def f():
     assert not s.counters["op"].components[0]["gather"]
 
 
+@pytest.mark.slow
```

Comment:
This test runs in ~1.1 seconds. I could not find the setting to tweak this timeout?